newEvaluation.html

```diff
@@ -57,7 +58,7 @@
 	<!-- Section3 -->
 	<section>
 		<h5>Should the candidate be hired?</h5>
-		<textarea rows = "5" cols = "" placeholder = "The type of project that is suitable for the candidate. Is guidance required for the candidate"></textarea>
+		<textarea rows = "5" cols = "" placeholder = "The type of project that is suitable for the candidate.&#10;Is guidance required for the candidate"></textarea>
```

Comment:
i used it to break row inside placeholder. I first tried <br> and /n but this were not working good because it were shown on the page.